api_specs/v2/component_analyses.yaml

```diff
@@ -118,76 +295,86 @@ components:
           items:
             type: object
             properties:
-              vendor_cve_ids:
+              cve_id:
                 type: string
+                example: "CVE-1234"
               cvss:
                 type: string
+                example: 9.2
               is_private:
                 type: boolean
+                example: true
 
     RegisteredUser:
-      title: Vendor Private Recommendation Information
-      description: Vendor Recommendation information for Premium Response.
+      title: Vendor Private Recommendation
       type: object
       properties:
         recommendation:
           type: object
           properties:
             recommended_version:
               type: string
-            affected_version:
-              type: string
+              example: 2
             component_analyses:
               $ref: '#/components/schemas/VulnerabilityDetails'
             message:
               type: string
-            severity:
+              example: "3 vulnerabilities within this dependency, 0 exploitable vulnerabilities"
+            highest_severity:
               type: string
+              example: "high"
             public_vulnerabilities_count:
               type: integer
+              example: 0
             private_vulnerabilities_count:
               type: integer
+              example: 0
             exploitable_vulnerabilities_count:
               type: integer
+              example: 0
             vendor_package_link:
               type: string
+              example : "https://snyk.io/vuln/maven:org.webjars%3Ajquery-form"
 
     FreeUser:
-      title: Vendor Public Recommendation Information
-      description: Vendor Recommendation information for Standard Response.
+      title: Vendor Public Recommendation
```

Comment:
You will need the package and version on each of them to be able to match them. If I get an array of these back how will I know which one maps to which package?

Reply:
Frontend needs to parse and map over the API result. 

I mean if the manifest file has EPV `django 1.1`, the developer needs to search it in API results for mapping.

Reply:
Do the API results have that info?

Reply:
+1 to @prashbnair . The result must have package name + version for batch calls at least.

Reply:
Ack!